gke-policies/policy/nap_forbid_default_sa.rego

```diff
@@ -0,0 +1,38 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# METADATA
+# title: Forbid default SA in NAP
+# description: NAP configuration should not allow default Service Accounts
+# custom:
+#   group: Security
+#   severity: Critical
+#   sccCategory: NAP_DEFAULT_SA_CONFIGURED
+#   cis:
+#     version: "1.2"
+#     id: "5.2.1"
```

Comment:
not sure if this is correct?

Reply:
Yes, this is same for default service accounts in general:
*"Ensure GKE clusters are not running using the Compute Engine default service account"*

For future, you can download GKE CIS Benchmark v1.2 from [CIS security page](https://www.cisecurity.org/cis-benchmarks/). GKE benchmark is under Kubernetes section so you need to expand to find ours :-)